src/main/java/data_access/ApiDataAccessObject.java

```diff
@@ -16,6 +16,8 @@
 import okhttp3.Response;
 import use_case.ApiDataAccessInterface;
 
+import entity.*;
```

Comment:
Checkstyle will complain that you should import a specific class. Check above for imports of specific Entities.